src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,105 @@
+package task;
+import formatter.TimeFormatter;
+import java.util.ArrayList;
+
+
+/**
+ *
+ */
+
+public class TaskList {
+    ArrayList<Task> myTaskList;
+
+    /**
+     *
+     */
+
+    public TaskList() {
+        myTaskList = new ArrayList<>();
+    }
+
+    /**
+     *
+     */
+
+    public ArrayList<Task> getList() {
+        return myTaskList;
+    }
+
+    public ArrayList<Task> searchTasks(String input) {
+        ArrayList<Task> answer = new ArrayList<>();
+
+        for(Task temp: myTaskList) {
+            if (temp.getName().contains(input)) {
+                answer.add(temp);
+            }
+        }
+        return answer;
+        }
+
+
+
+    /**
+     *
+     */
+
+    //Add and print the added notes
```

Comment:
Same as a previous comment of mine: perhaps the common chunks of code can be broken down into smaller methods?